dictionary/README.md

```diff
@@ -2,4 +2,6 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
+Restart strategy should be :transient so that the child process is restarted only if it terminate abnormally
```

Comment:
# -3

For the Dictionary I'd like to see :permanent. If it exits for any reason, I'd like it restarted so games can continue.

Now, there's no reason it _should_ exit, but :permanent expresses intent better than :transient